src/ast/value.rs

```diff
@@ -193,6 +196,46 @@ pub fn escape_single_quote_string(s: &str) -> EscapeSingleQuoteString<'_> {
     EscapeSingleQuoteString(s)
 }
 
+pub struct EscapeEscapedStringLiteral<'a>(&'a str);
+
+impl<'a> fmt::Display for EscapeEscapedStringLiteral<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut is_escaped = true;
```

Comment:
>  don't see is_escaped ever set to false -- I would expect it would start with is_escaped = true and then is_escaped wold be set to false after each character was written

I will remove it, It's useles.

It's not a correct value, because the last `\` should be escaped.
<img width="664" alt="image" src="https://user-images.githubusercontent.com/572096/170065798-a620fad1-28bf-4169-965a-2c78402a6eca.png">